configure

```diff
@@ -5003,7 +5070,11 @@ EOF
 	cc_check $ZLIB_CFLAGS $ZLIB_LIBS -lz && _zlib=yes
 fi
 if test "$_zlib" = yes ; then
-	append_var ZLIB_LIBS "-lz"
+	if test "$_host_os" != kolibrios ; then
+		append_var ZLIB_LIBS "-lz"
+	else
+		append_var ZLIB_LIBS "-lz.dll"
```

Comment:
Ugh, this is ugly

Reply:
I don't know how to do it better

Reply:
If you're cross-compiling or otherwise have access to pkg-config, what could work is to create a directory in the ScummVM repository with basic .pc files and append that to the usual search path, then use that for detecting all libraries that are known to provide .pc files normally.

Reply:
Doesn't help. "-lz" is hardcoded. And for others creating .pc in advance is impossible as KOS32_* variable may change. So we have to preprocess them which is as annoying as current solution with having the paths in configure

Reply:
See what I proposed for `-ljpeg`

For your `.pc` relocation, you need a kos32-pkg-config wrapper making use of KOS32_ variable to call real pkg-config with proper environment variables. It's how it's done in many toolchains I saw.
Look [here](https://github.com/devkitPro/pacman-packages/blob/master/gba/pkg-config/pkg-config.in) for an example.
But it's more the job of a toolchain than ours...